environments/prod/mailrelay-platform-hmcts-internal.yml

```diff
@@ -0,0 +1,39 @@
+name: "mailrelay.internal.platform.hmcts.net"
```

Comment:
also needs adding to https://github.com/hmcts/azure-private-dns/tree/master/components/prod